### PR TITLE
Create permission file for maven-license-plugin

### PR DESCRIPTION
**Pull request by NotMyFault**
For https://github.com/jenkinsci/maven-license-plugin/

### CODE REVIEW

**Review by timja — APPROVED** [review]
may need an update in https://github.com/jenkins-infra/repository-permissions-updater/blob/master/permissions/component-kohsuke-maven-plugins.yml as well

/ an additional path here

**Review by timja — APPROVED** [review]

File: permissions/component-kohsuke-maven-plugins.yml [hunk 1]
```diff
@@ -4,5 +4,6 @@ name: "dummy-unused-artifact-name"
 paths:
   - "org/jvnet"
   - "org/kohsuke"
+  - "io/jenkins/tools/maven"
```
Comment (timja) [inline permissions/component-kohsuke-maven-plugins.yml:7]: might need to move this to `permissions/component-license-maven-plugin.yml` ? as won't it need to be cd enabled?

Comment (NotMyFault) [reply permissions/component-kohsuke-maven-plugins.yml:7]: Doesn't "- "io/jenkins/tools/maven/license-maven-plugin"" suffice in the license-maven-plugin.yml?

Comment (timja) [reply permissions/component-kohsuke-maven-plugins.yml:7]: no maven plugins also update the groupid IIRC

**Review by timja — APPROVED** [review]

**Pull request merged**